hasktorch/src/Torch/Typed/Native.hs

```diff
@@ -311,6 +311,20 @@ selu
   -> Tensor device dtype shape -- ^ output
 selu input = unsafePerformIO $ cast1 ATen.selu_t input
 
+-- | mish
+-- `mish` is a smooth activation function, see https://arxiv.org/abs/1908.08681 for details.
+-- >>> dtype &&& shape &&& (\t -> D.asValue (toDynamic t) :: [[Float]]) $ mish (ones :: CPUTensor 'D.Float '[3,2])
+-- (Float,([3,2],[[0.86509836,0.86509836],[0.86509836,0.86509836],[0.86509836,0.86509836]]))
+mish
+  :: forall shape dtype device
+   . ( StandardFloatingPointDTypeValidation device dtype
+     , BasicArithmeticDTypeIsValid device dtype
+     , shape ~ Broadcast shape shape
+     )
+  => Tensor device dtype shape
+  -> Tensor device dtype shape
+mish = mul =<< tanh . softplus (1 :: Float) 20
```

Comment:
Both 1 and 20 are just default values of softplus.

Reply:
yes, that's right

Reply:
I don't think @digantamisra98 specified what these should be in his paper. Thus, I went with the default.

Reply:
nvm, he defined `softplus x = ln (1 + e^x)`, so that settles it.

Reply:
Oh, should I revert this merge?

Reply:
I'm sorry to merge early.

Reply:
No, it’s fine the way it is. Thank you!